internal/txapp/txapp.go

```diff
@@ -490,7 +494,8 @@ func (r *TxApp) processVotes(ctx context.Context, db sql.DB, block *common.Block
 
 	for _, resolution := range expired {
 		expiredIDs = append(expiredIDs, resolution.ID)
-		if resolution.Type != voting.ValidatorJoinEventType && resolution.Type != voting.ValidatorRemoveEventType {
+		if resolution.Type != voting.ValidatorJoinEventType && resolution.Type != voting.ValidatorRemoveEventType &&
```

Comment:
What about the `range finalized` above?  

What's the common characteristic between these event types and how we handle them in expired and finalized?  If there's a clear quality, perhaps the a helper function like `isOngoingEventType` or whaterver it is we are after.

 We're definitely going to forget to update this when new event types are added.